xenial_pdi/Dockerfile

```diff
@@ -15,5 +15,6 @@ USER 0
 
 COPY --from=builder /usr/local /usr/local
 RUN echo ". /usr/local/share/pdi/env.bash" > /etc/profile.d/zzz_enable_pdi.sh
+ && apt install hdf5-tools -y
```

Comment:
Can you follow the pattern used in other Dockerfiles for apt calls:
```
 && apt-get update -y \
 && apt-get upgrade -y \
 && apt-get install -y --no-install-recommends \
    your-packages-here \
 && apt-get autoremove -y \
 && apt-get clean -y \
 && apt-get autoclean -y \
 && rm -rf /var/lib/apt/lists/*
```

* `apt-get` ensure a scriptable interface unlike `apt`
* the initial `update` & `upgrade` to ensure the install works,
* the `--no-install-recommends` to ensure the minimum number of packages is installed
* the `autoremove`, `clean`, `autoclean` & `rm` ensure the image size is minimal